src/lavinmq/parameter_store.cr

```diff
@@ -76,6 +76,9 @@ module LavinMQ
         end
       end
       @log.debug { "#{size} items loaded from #{@file_name}" }
+    rescue ex
+      @log.error { "Failed to load #{@file_name}: #{ex}" }
```

Comment:
Maybe
```suggestion
      @log.error(exception: ex) { "Failed to load #{@file_name}" }
```
Or dunno how much we want to log, but I guess the more the merrier in a "rescue all" clause?

Reply:
yes! nice

Reply:
And include the exception in the other rescues as well I suppose? :)

Reply:
yes, got stuck in other stuff but will fix the rest too!